### PR TITLE
adds deployment for render.com

### PR DESCRIPTION
**Pull request by phishy**
## What does this PR do?

Adds a one-click deployment button for [Render.com](https://render.com/)

**Note**:

During my investigation of Heroku, Render, and user messages on Slack, Cal.com is incredibly memory intensive to build. This makes it difficult to use cheap compute to build the app on. It makes the most sense to build a docker image and deploy it to these destinations, until the cal build process streamlines.

Thus, for Render, I had to make another repo that references the `calcom/docker` repo so that it would be prebuilt. The same will have to be done for Heroku, and the size is doubly too large for non-Docker Heroku deployments.

## Type of change

- [x] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- [ ] Click [Deploy to Render](https://render.com/deploy?repo=https://github.com/phishy/docker-render)
- [ ] Check Cal.com installation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,6 +293,12 @@ Currently Vercel Pro Plan is required to be able to Deploy this application with
 
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fcalcom%2Fcal.com&env=DATABASE_URL,NEXT_PUBLIC_WEBAPP_URL,NEXTAUTH_URL,NEXTAUTH_SECRET,CRON_API_KEY,CALENDSO_ENCRYPTION_KEY&envDescription=See%20all%20available%20env%20vars&envLink=https%3A%2F%2Fgithub.com%2Fcalcom%2Fcal.com%2Fblob%2Fmain%2F.env.example&project-name=cal&repo-name=cal.com&build-command=cd%20../..%20%26%26%20yarn%20build&root-directory=apps%2Fweb%2F)
 
+### Render
+
+<a href="https://render.com/deploy?repo=https://github.com/calcom/cal.com/tree/render-deploy">
```
Comment (phishy) [inline README.md:298]: After this is approved, this must be changed to remove the branch.

File: render.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+services:
+  - type: web
+    name: cal-web
+    plan: pro plus
+    repo: https://github.com/phishy/cal.com.git
```
Comment (phishy) [inline render.yaml:5]: This URL will need to be changed after successful review.

**Comment by PeerRich** [comment]
curious: does render come with a postgres or does it only host frontends like vercel?

File: README.md [hunk 3]
```diff
@@ -293,6 +293,10 @@ Currently Vercel Pro Plan is required to be able to Deploy this application with
 
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fcalcom%2Fcal.com&env=DATABASE_URL,NEXT_PUBLIC_WEBAPP_URL,NEXTAUTH_URL,NEXTAUTH_SECRET,CRON_API_KEY,CALENDSO_ENCRYPTION_KEY&envDescription=See%20all%20available%20env%20vars&envLink=https%3A%2F%2Fgithub.com%2Fcalcom%2Fcal.com%2Fblob%2Fmain%2F.env.example&project-name=cal&repo-name=cal.com&build-command=cd%20../..%20%26%26%20yarn%20build&root-directory=apps%2Fweb%2F)
 
+### Render
+
+[![Deploy to Render](https://render.com/images/deploy-to-render-button.svg)](https://render.com/deploy?repo=https://github.com/phishy/docker-render)
```
Comment (PeerRich) [inline README.md:298]: should this be /phishy/?

Comment (PeerRich) [reply README.md:298]: we can open a new repo if you want under calcom/docker-render or something

Comment (phishy) [reply README.md:298]: Exactly

Comment (PeerRich) [reply README.md:298]: or should we put it in calcom/docker? https://github.com/calcom/docker

Comment (PeerRich) [inline README.md:298]:
```suggestion
[![Deploy to Render](https://render.com/images/deploy-to-render-button.svg)](https://render.com/deploy?repo=https://github.com/calcom/docker-render)
```

Comment (phishy) [reply README.md:298]: Actually I think that might work. I'll submit a PR there.

Comment (PeerRich) [reply README.md:298]: cool. let me know and i will archive the new repo I just made

**Pull request closed without merging**